### PR TITLE
✨ NavigationBar component

### PR DESCRIPTION
**Pull request by manusmd**
# What I did:
- created navigation bar on base of figma design with minor color changes
- implemented nav icons as JSX Elements to make the appearance dynamic with the fill prop
- added different styles for each page to storybook
- implemented react router

# Design
## Home
image

## Students
image

## Teachers
image

## Groups
image

## Subjects
image

Part of #4

### CODE REVIEW

**Review by davidwobido — APPROVED** [review]
🚀  Very nice. Just one styling-opinion: maybe you could dim the shadow of the home button a bit. It’s very strong atm.

**Review by JulianSchumacher99 — APPROVED** [review]
🚀

File: src/Components/NavigationBar/NavigationBar.module.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.navigationBar {
+  display: grid;
+  width: 100%;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```
Comment (lajanzen) [inline src/Components/NavigationBar/NavigationBar.module.css]:
```suggestion
  grid-template-columns: repeat(5, 1fr);
```

File: src/Components/NavigationBar/assets/Teachers.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export default function TeachersIcon(
+  props: React.SVGProps<SVGPathElement>
+): JSX.Element {
+  return (
+    <svg
+      width="39"
+      height="36"
+      viewBox="0 0 39 36"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (lajanzen) [inline src/Components/NavigationBar/assets/Teachers.tsx]:
```suggestion
```

File: src/Components/NavigationBar/assets/Subjects.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export default function SubjectsIcon(
+  props: React.SVGProps<SVGPathElement>
+): JSX.Element {
+  return (
+    <svg
+      width="36"
+      height="36"
+      viewBox="0 0 36 36"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (lajanzen) [inline src/Components/NavigationBar/assets/Subjects.tsx]:
```suggestion
```

File: src/Components/NavigationBar/assets/Students.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+export default function StudentsIcon(
+  props: React.SVGProps<SVGPathElement>
+): JSX.Element {
+  return (
+    <svg
+      width="32"
+      height="35"
+      viewBox="0 0 32 35"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (lajanzen) [inline src/Components/NavigationBar/assets/Students.tsx]:
```suggestion
```

File: src/Components/NavigationBar/assets/Home.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+export default function HomeIcon(): JSX.Element {
+  return (
+    <svg
+      width="39"
+      height="33"
+      viewBox="0 0 39 33"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (lajanzen) [inline src/Components/NavigationBar/assets/Home.tsx]:
```suggestion
```

File: src/Components/NavigationBar/assets/Groups.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+export default function GroupsIcon(
+  props: React.SVGProps<SVGPathElement>
+): JSX.Element {
+  return (
+    <svg
+      width="44"
+      height="28"
+      viewBox="0 0 44 28"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (lajanzen) [inline src/Components/NavigationBar/assets/Groups.tsx]:
```suggestion
```

**Review by lajanzen — COMMENTED** [review]
🚀

**Pull request merged**